src/view/pane/message.js

```diff
@@ -110,8 +110,20 @@ Candy.View.Pane = (function(self, $) {
         timestamp = Candy.Util.iso8601toDate(timestamp);
       }
 
+      var messagePane = self.Room.getPane(roomJid, ".message-pane");
```

Comment:
can we rely on the fact that each client behaves like it should & generates a unique id for the message, also that not two clients generate the same id for different messages?